### DISCUSSION TITLE
Repository not showing in repositories

### DISCUSSION THREAD

**Discussion by ashirmustafa** [community/community#134113] [Repositories] [answered]
### Select Topic Area

Question

### Body

Someone added me to their repo through email and i have access to it, i can open it using link but cannot see it in my repos list. i've tried filtering private/public too. and for this reason i think my contributions aren't being counted too.
I've also tried git config user.email <email> and it's correct.

**Comment by davevad93** [comment, marked as answer]
Hi @ashirmustafa ,

In the `Repositories` section, only your own or forked repos are listed, to access repos owned by others that you have access to, go to the top left section of the dashboard and locate them with the search bar:

![0](https://github.com/user-attachments/assets/cef6906a-7fd1-4ee3-a54d-297967f83898)

About contributions not being counted, in case it's a `private` repo make sure you have the following option checked in your [profile settings](https://github.com/settings/profile):

![1](https://github.com/user-attachments/assets/420a1129-7805-44f5-ae37-42996707f6e1)

Also note that only commits made in the `main` branch are counted as contributions, commits from secondary branches must be merged into the main branch to be counted.

**Comment by NimsaraLiyanage** [comment]


**Reply by DinLiren** [reply]
@NimsaraLiyanage Thank you

**Comment by AllenCohn** [comment]
_Very new_ user here... I have what I _think_ is a related problem...not all of the repositories that I _think_ I am attached to show up in the drop down list. There is one other public repository that I forked from... I want it to show up in this list, but it doesn't. (The very first one listed...PCGen/pcgen...is a public repository that I forked, and it does show up. I don't know why that one shows up, but the other public repository doesn't.)

Thanks, Allen
![image](https://github.com/user-attachments/assets/a741e263-1012-4e43-beb5-b9947ed3de1e)
